.readthedocs.yaml

```diff
@@ -0,0 +1,10 @@
+version: 2
+sphinx:
+  configuration: ./bindings/python/docs/source/conf.py
+conda:
+  environment: ./bindings/python/docs/environment.yml
+python:
+  install:
+    # install the package itself
+    - method: pip
+      path: ./bindings/python
```

Comment:
Does this build properly properly on readthedocs now that the workarounds are removed? If so do we need jupyterlab at all?

Reply:
We're only using JupyterLab Maintainer-Tools to simplify the Actions logic, it saves us from having to keep all those things up to date and in sync.

Reply:
The PR build works!

Reply:
Removed usage of the jupyterlab action